.github/workflows/examples.yml

```diff
@@ -0,0 +1,28 @@
+name: examples
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  examples:
+    name: Build and commit examples
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Build examples and docs
+      run: |
+        make examples
+        make docs
```

Comment:
I think we'll also need to add these to run before the Netlify preview is generated, otherwise they won't be reflected. Currently for this repo, we're using the build pipeline that Netlify hosts itself, so we'd have to configure it to happen in there.

Reply:
done